<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersDialog</class>
 <widget class="QDialog" name="UsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочник пользователей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/Image/users32.png</normaloff>:/Image/users32.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Новый</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/Image/iconfinder_users-add_44502.png</normaloff>:/Image/iconfinder_users-add_44502.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="text">
        <string>Правка</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/Image/usersedit.png</normaloff>:/Image/usersedit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/Image/exit.png</normaloff>:/Image/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxUserData">
     <property name="title">
      <string>Данные пользователя</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLogin">
        <property name="text">
         <string>Логин:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLogin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPass"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxIsActive">
        <property name="text">
         <string>Активный</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Список пользователей</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
